examples/icmp/k8s/simple-client.yaml

```diff
@@ -19,4 +19,4 @@ metadata:
   name: "simple-client"
   namespace: default
   annotations:
-    ns.networkservicemesh.io: icmp-responder?app=simple-client
+    ns.networkservicemesh.io: icmp-responder?app=simple-client&ifname=responder0
```

Comment:
Could we use:

icmp-responder/responder0?app=simple-client

as speced out here:

https://github.com/networkservicemesh/networkservicemesh/issues/708